src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -166,7 +166,7 @@ public void Import(TrackedDownload trackedDownload)
 
         public bool VerifyImport(TrackedDownload trackedDownload, List<ImportResult> importResults)
         {
-            var allTracksImported = importResults.All(c => c.Result == ImportResultType.Imported) ||
+            var allTracksImported = (importResults.Any() && importResults.All(c => c.Result == ImportResultType.Imported)) ||
```

Comment:
Is there a reason why .All needs to be checked? It seems like this would cause a partial import while files are still being synced in the case of remote path mapping. (i.e. 10 tracks in album, 1 gets synced, then import succeeds with the 1 track).

Reply:
Sorry I meant the .All check. It seems like the .Any was added to handle an empty import list, since .All will return true for a result set >= 0 in length. And should partial imports even be taken into account for automatic imports? It's my understanding that an automatic import will fail if the downloaded files do not match the selected release. At least files are flagged as such during a manual import, if the files do not match what is expected.